vm/interpreter/src/gas_block_store.rs

```diff
@@ -1,15 +1,29 @@
 // Copyright 2019-2022 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
-use super::gas_tracker::{GasTracker, PriceList};
+use super::gas_tracker::PriceList;
 use cid::{Cid, Code};
 use db::{Error, Store};
 use forest_encoding::{de::DeserializeOwned, ser::Serialize, to_vec};
+use fvm::gas::{GasCharge, GasTracker};
+use fvm::kernel::ExecutionError;
 use ipld_blockstore::BlockStore;
 use std::cell::RefCell;
 use std::error::Error as StdError;
 use std::rc::Rc;
 
+pub fn to_std_error(exec_error: ExecutionError) -> Box<dyn StdError> {
```

Comment:
Could you add `// FIXME: remove when error handling has moved to anyhow::Error. Tracking issue: https://github.com/ChainSafe/forest/issues/1536` ?

Reply:
Done